tests/default_vars.sh

```diff
@@ -614,11 +614,11 @@ export_35d_run ()
 export CNTL_DIR=""
 export LIST_FILES=""
 }
-export_datm ()
+export_datm_cdeps_cfsr ()
```

Comment:
I think there is no need to add defaults for every different streams supported by the CDEPS. A common variables can be defined in the defaults and specialized one can be done in the file under tests/tests. I am following this convention in the HAFS to define HAFS specific RTs.

Reply:
@uturuncoglu The name is changed to export_datm_cdeps.